test/warn-decide-predicate-unknown/lemmas-no-smt-spec.stderr.out.golden

```diff
@@ -0,0 +1,132 @@
+kore-exec:  Warning (DecidePredicateUnknown):
```

Comment:
@ana-pantilie I've modified the integration test, to instead comapre against the stderr output